### PR TITLE
Updated info to add PGN mainnet and testnet

### PR DESCRIPTION
**Pull request by sodofi**
Corrected the addresses. Please let me know if this needs updating or if there is more information needed from PGN.

### CODE REVIEW

File: .history/src/assets/v1.3.0/compatibility_fallback_handler_20231102170232.json [hunk 1]
```diff
@@ -0,0 +1,478 @@
+{
```
Comment (mmv08) [inline .history/src/assets/v1.3.0/compatibility_fallback_handler_20231102170232.json:1]: What are those files? Please remove them

File: src/assets/v1.3.0/compatibility_fallback_handler.json [hunk 2]
```diff
@@ -123,6 +124,7 @@
     "54211": "0x017062a1dE2FE6b99BE3d9d37841FeD19F573804",
     "56288": "0x017062a1dE2FE6b99BE3d9d37841FeD19F573804",
     "57000": "0x017062a1dE2FE6b99BE3d9d37841FeD19F573804",
+    "58008": "0x1AC114C2099aFAf5261731655Dc6c306bFcd4Dbd",
```
Comment (mmv08) [inline src/assets/v1.3.0/compatibility_fallback_handler.json:127]: this address doesn't match the expect address

File: src/assets/v1.3.0/sign_message_lib.json [hunk 3]
```diff
@@ -51,6 +51,7 @@
     "336": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "338": "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
     "420": "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
+    "424": "0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4",
```
Comment (mmv08) [inline src/assets/v1.3.0/sign_message_lib.json:54]: This address doesn't match the expected one

**Review by mmv08 — CHANGES_REQUESTED** [review]

File: .history/src/assets/v1.3.0/compatibility_fallback_handler_20231102170232.json (hunk 1 above)
Comment (sodew) [reply .history/src/assets/v1.3.0/compatibility_fallback_handler_20231102170232.json]: These were removed

**Comment by sodew** [comment]
@mmv08 

Fixed PR so that .history files are removed, and mainnet contracts match expected addresses.

Additionally, removed testnet for now since some of the contracts aren't verified on PGNs end.

**Comment by mmv08** [comment]
> Additionally, removed testnet for now since some of the contracts aren't verified on PGNs end.

It is still present in one of the files. Also, it's okay with us if the contract isn't verified since the deployment is deterministic and depends on the hash of the bytecode. If one were to alter the code, it would result in a different address.
https://github.com/safe-global/safe-deployments/pull/315/files#diff-905275e3e6a99d588ff498e27b13efa9ee12d1ae314645f224cecca748ed98feR126

**Comment by sodew** [comment]
@mmv08 Just added back testnet with expected contract addresses. 

Is there more we need to ensure is done on the PGN side to ensure this PR gets merged?

**Review by mmv08 — APPROVED** [review]

**Pull request merged**